hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -98,15 +98,21 @@
 
   // Add run modes.
   run_modes: [
-    {
-      name: sw_test_mode
-      sw_images: ["sw/device/boot_rom/boot_rom:0"]
+    { // This base mode initializes the OTP / life cycle with the DEV state
```

Comment:
Glad to see this method is working ok to extend to other types of images that need to be preloaded. Also, this consolidation looks great. 

nit: I will probably add `otp` somewhere in `base_lc_dev_mode` name.

Reply:
will do!

Reply:
yes the integration was really convenient and smooth.